pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -100,10 +100,12 @@ buildPythonApplication rec {
   CGO_ENABLED = 0;
   GOFLAGS = "-trimpath";
 
+  # allow for both overrideAttrs and vendorHash updating
   goModules = (buildGoModule {
     pname = "kitty-go-modules";
     inherit src vendorHash version;
```

Comment:
```suggestion
    inherit src version;
    vendorHash = "sha256-jk2EcYVuhV/UQfHAIfpnn8ZIZnwjA/o8YRXmpoC85Vc=";
```

Reply:
I made this change, but also moved `goModules` up by the source so it's easier to see if someone wants to override it.